src/preact/base-element.js

```diff
@@ -222,6 +222,7 @@ export class PreactBaseElement extends AMP.BaseElement {
       ...childrenInit,
       ...passthroughInit,
       ...templatesInit,
+      ...this.getAdditionalMutationObserverInitProperties(),
     });
```

Comment:
So now the init from the amp layer is mixed with the init for the generic mutation observer.  Hmm... does it still make sense to use the base-element Mutation Observer then?